src/kademlia/dht_tracker.cpp

```diff
@@ -238,53 +238,10 @@ namespace libtorrent { namespace dht
 		m_dht.announce(ih, listen_port, flags, f);
 	}
 
-	namespace {
-
-	// these functions provide a slightly higher level
-	// interface to the get/put functionality in the DHT
-	bool get_immutable_item_callback(item& it, boost::function<void(item const&)> f)
```

Comment:
@thomas-yuan would you mind putting these back (even if being redundant) to not collide too badly with ssiloti's patch?